ilmercatinodellibro/ilmercatinodellibro
# feat(server&client): add FAQ

## What it does
This feature fixes this issue:
- #56 

## How to test it
Select a location, on the login page you will see the FAQ of the selected retail location
Log in and press the "FAQ" button on the drawer/top bar
